recipes/mne-python_1.0/construct.yaml

```diff
@@ -33,8 +35,8 @@ post_install: ../../assets/post_install_windows.bat     # [win]
 
 # Create signed macOS .pkg installer
 installer_type: pkg                     # [osx]
-signing_identity_name: 9779L28NP8       # [osx]  Used for productsign
-notarization_identity_name: 9779L28NP8  # [osx]  Used for codesign
+signing_identity_name:                  # [osx]  Used for productsign
+notarization_identity_name:             # [osx]  Used for codesign
```

Comment:
This should be reverted for signing on CI, no?

Reply:
See changes in CI. Rather than always have it in and remove for pull requests, I do the opposite and always have it blank but add it for non-PRs (main/release builds). This makes it work better for local builds as a useful side effect.